docs/csharp/csharp-7.md

```diff
@@ -16,64 +16,539 @@ ms.assetid: fd41596d-d0c2-4816-b94d-c4d00a5d0243
 # What's new in C# 7
 
 C# 7 adds a number of new features to the C# language:
-
 * [`out` variables](#out-variables)
```

Comment:
It looks strange to me to see language keyword styling in a list or in a heading. I would expect to see the style applied within paragraphs, but not elsewhere. Should 'out' and other keywords in this article be styled as plain text?